25825.8 <c-14#77> Created logger
25825.8 <c-14#77> appl_initialize was called
25825.8 <c-14#77:30> Logger was updated
25825.8 <c-14#77:30> Started intializing cube lib
25825.8 <c-14#77:30> Finished intializing cube lib with worker, starting to work
25825.9 <c-14#77:30> requestCubes => [51] Buffer size: 0
25825.9 <c-14#77:30> Sent requestCubes signal to root
25826 <c-14#77:30> Received 1 cubes from root
25826 <c-14#77:30> The main loop continues
25826 <c-14#77:30> Started solving a cube
25871.1 <c-14#77:30> Cube failed
25871.1 <c-14#77:30> Added failed cube
25871.1 <c-14#77:30> returnFailedCubes => [51] Buffer size: 3
25871.1 <c-14#77:30> Sent 1 failed cubes to root
25871.1 <c-14#77:30> Received receivedFailedCubes signal from root
25871.1 <c-14#77:30> requestCubes => [51] Buffer size: 0
25871.1 <c-14#77:30> Sent requestCubes signal to root
25871.1 <c-14#77:30> Received 1 cubes from root
25871.1 <c-14#77:30> The main loop continues
25871.1 <c-14#77:30> Started solving a cube
26752.6 <c-14#77:30> appl_interrupt was called
26752.6 <c-14#77:30> Interrupting worker
26752.6 <c-14#77:30> appl_interrupt was called
26752.6 <c-14#77:30> appl_withdraw was called
26752.6 <c-14#77:30> STOP (0.00s since last cb)
26752.6 <c-14#77:30> Solving interrupted
26752.6 <c-14#77:30> The main loop continues
26752.6 <c-14#77:30> Leaving the main loop due to an interruption
26752.6 <c-14#77:30> Started withdraw thread
26752.6 <c-14#77:30> Finished withdrawing lib
26752.6 <c-14#77:30> Enter destructor of CubeWorker
26752.6 <c-14#77:30> Time waiting for messages: 0.000
26752.7 <c-14#77:30> Enter destructor of CubeWorkerInterface
26752.7 <c-14#77:30> Enter destructor of CubeRoot
26752.7 <c-14#77:30> Finished withdraw thread
26815.5 <c-14#77:30> Enter destructor
26815.5 <c-14#77:30> Time working: 926.752
26815.5 <c-14#77:30> Time suspended: 0.000
26815.5 <c-14#77:30> Joined cleanup thread
26815.5 <c-14#77:30> Exit destructor
26815.5 <c-14#77:30> Destructing logger
